deploy/helm/kubecf/values.yaml

```diff
@@ -21,24 +21,25 @@ releases:
   defaults:
     url: docker.io/cfcontainerization
     stemcell:
-      os: opensuse-42.3
-      version: 36.g03b4653-30.80-7.0.0_362.g9610e90b
+      os: sle15-15SP1
+      version: 5.g4792c5a-0.2-7.0.0_362.g9610e90b
+
   bits-service:
     stemcell:
+      os: opensuse-42.3
       version: 36.g03b4653-30.80-7.0.0_348.gc8fb3864
-  capi:
-    version: 1.83.0
-    stemcell:
-      version: 36.g03b4653-30.80-7.0.0_360.g0ec8d681
   cf-acceptance-tests:
     stemcell:
+      os: opensuse-42.3
```

Comment:
IIRC, CATS only had a custom stemcell because the default one didn't have a CATS image.  If we're switching to a newer stemcell we can probably drop the custom stemcell (just delete the block, the helm templating should be smart enough to pick up the default one).

Please also check the other releases to see if they can be switched to the default stemcell.

Reply:
@mook-as this sounds like it should be another ticket
@aduffeck - did you try with the default and it failed?

Reply:
@mook-as @viovanov I switched the CATS to the SLE15 based stemcell in 63c4f76. The remaining opensuse stemcells are currently required and will be tackled separately later on.